src/components/Search/Search.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from "react";
+import { StyledAsyncSelect } from "./Search.styles";
+import { getAllCoins } from "../../store/search/searchActions";
+
+const Search = ({ allCoins, isLoading }) => {
+  const [value, setValue] = useState("");
+
+  useEffect(() => {
+    getAllCoins(value);
+  }, [getAllCoins, value]);
```

Comment:
const [data, setData] = useState([]);

```suggestion
  useEffect(async () => {
    const data = await axios(`${base}/${inputValue}`);
    setData(data)
  }, [value]);
```